### ISSUE TITLE
Improve the cursor positions that enable 'Pull up to interface'

### ISSUE THREAD

**Issue by Tragetaschen** [dotnet/roslyn#40293]
**Version Used**: 
16.4.1

**Steps to Reproduce**:
```c#
#nullable enable
using System;

public interface ITest
{
}

public class Test : ITest
{
    public event Action? E; // Try to activate here
}
```

**Expected Behavior**:
Putting the cursor before and after the `E` should activate the 'Pull member up to interface' refactoring.

**Actual Behavior**:
Putting it after the `E` IntelliSense-highlights the member, but doesn't trigger the refactoring

**Comment by CyrusNajmabadi** [comment]
Closing out due to lack of feedback.
